src/pages/book/components/complete/Complete.tsx

```diff
@@ -0,0 +1,49 @@
+import { NAVIGATION_STATE } from "@constants/navigationState";
+import { useHeader } from "@hooks/useHeader";
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import FreeBook from "../freeBook/FreeBook";
+import PaidBook from "../paidBook/PaidBook";
+
+const Complete = () => {
+  const navigation = useNavigate();
+  const isPaid = false;
+
+  // TODO: state로 받아오기
+  const title = "비트 정기공연";
+  const bankName = "농협";
+  const accountNumber = "3561202376833";
+  const price = 10000;
+  const id = 1;
+
+  const { setHeader } = useHeader();
+
+  useEffect(() => {
+    setHeader({
+      headerStyle: NAVIGATION_STATE.ICON,
+      title: "내가 등록한 공연",
+      subText: "삭제",
+      rightOnClick: () => {
+        navigation("/");
```

Comment:
p3) 저희 /에는 온보딩 넣고 main으로 하기로 해서!!!! 아마두,,,

```suggestion
        navigation("/main");
```

으로 바꿔주는 게 좋을 것 같아요!